pkgs/development/python-modules/libarcus/default.nix

```diff
@@ -20,8 +20,7 @@ buildPythonPackage rec {
   buildInputs = [ protobuf ];
 
   postPatch = ''
-    # To workaround buggy SIP detection which overrides PYTHONPATH
-    sed -i '/SET(ENV{PYTHONPATH}/d' cmake/FindSIP.cmake
+    sed -i 's#''${Python3_SITELIB}#${placeholder "out"}/${python.sitePackages}#' cmake/SIPMacros.cmake
```

Comment:
@FRidh Do you know the proper way to get the python site-packages directory in cmake?

Reply:
I don't think there is a better way short of patching CMake's `FindPython`, and even that is questionable because it uses standard Python's `sysconfig` to get these paths and some packages may (rightfully) expect `SITELIB` to be Python's core site-packages. The root of the problem is that Python on FHS systems has only one true `site-packages` and everything expects that.